commit dffce107fb68e21cd85938d67115bdafe4e697eb
Author: Sebastian Wagner <swagner@suse.com>
Date:   Mon Jul 22 14:36:41 2019 +0200

    mgr: cleanup idle debug log at level 4.
    
    Changed all messages to 10 that are printed repeatedly in an idle cluster.
    
    This makes debugging Python modules easier as the log
    is no longer pullued by mgr core messages.
    
    Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index 5726dcda424..0ee9dff4064 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -520,7 +520,7 @@ void Mgr::handle_mon_map()
 
 bool Mgr::ms_dispatch2(const ref_t<Message>& m)
 {
-  dout(4) << *m << dendl;
+  dout(10) << *m << dendl;
   std::lock_guard l(lock);
 
   switch (m->get_type()) {
